js/src/index.js

```diff
@@ -54,16 +54,15 @@ if (process.env.NODE_ENV === 'development') {
 }
 
 const AUTH_HASH = '#/auth?';
-const parityUrl = process.env.PARITY_URL || window.location.host;
-const urlScheme = window.location.href.match(/^https/) ? 'wss://' : 'ws://';
+const parityUrl = process.env.PARITY_URL || '127.0.0.1:8546';
 
 let token = null;
 
 if (window.location.hash && window.location.hash.indexOf(AUTH_HASH) === 0) {
   token = qs.parse(window.location.hash.substr(AUTH_HASH.length)).token;
 }
 
-const api = new SecureApi(`${urlScheme}${parityUrl}`, token);
+const api = new SecureApi(parityUrl, token);
```

Comment:
Hmm, right. So it's possible to override (in constructor) `protocol` returned by `SecureApi` in case server protocol should be different than the one the UI is running. I can refactor it and detect if protocol was specified in the URL and in such case always use that one if you prefer.